DOCS/man/options.rst

```diff
@@ -2447,6 +2447,10 @@ Subtitles
         This affects ASS subtitles as well, and may lead to incorrect subtitle
         rendering. Use with care, or use ``--sub-font-size`` instead.
 
+``--sub-scale-preserve-signs=<yes|no>``
```

Comment:
needs interface changes entry to explain how this changes this option and the other one